app/views/projects/samples/transfers/_dialog.html.erb

```diff
@@ -1,29 +1,89 @@
+        url: list_namespace_project_samples_path,
+        data: { "infinite-scroll-target": "pageForm" }
+      ) do %>
+        <div data-infinite-scroll-target="pageFormContent"></div>
+      <% end %>
+      <div
+        class="overflow-y-auto max-h-[300px] mb-4"
+        data-action="scroll->infinite-scroll#scroll"
+        data-infinite-scroll-target="scrollable"
+      >
+        <%= turbo_frame_tag "list_select_samples" do %>
+          <div role="status" class="max-w-sm animate-pulse">
+            <% 4.times do %>
+              <div class="h-2 bg-gray-200 rounded-full dark:bg-gray-700 max-w-[360px] mb-2.5"></div>
+              <div class="h-2 bg-gray-200 rounded-full dark:bg-gray-700 mb-2.5"></div>
+              <div class="h-2 bg-gray-200 rounded-full dark:bg-gray-700 max-w-[330px] mb-2.5"></div>
+              <div class="h-2 bg-gray-200 rounded-full dark:bg-gray-700 max-w-[300px] mb-2.5"></div>
+              <div class="h-2 bg-gray-200 rounded-full dark:bg-gray-700 max-w-[360px] mb-2.5"></div>
+            <% end %>
+            <span class="sr-only"><%= t(:".loading") %></span>
           </div>
-          <div>
-            <%= form.submit t(".submit_button"),
-                        class: "button button--size-default button--state-primary",
-                        disabled: @projects.count.zero?,
-                        data: {
-                          action: "click->sessionstorage-amend-form#clear"
-                        } %>
+        <% end %>
+      </div>
+
+      <%= turbo_frame_tag "transfer_samples_dialog_content" do %>
+        <%= form_for(:transfer, url: namespace_project_samples_transfer_path, method: :post,
+              data: {
+                controller: "spinner",
+                action:"turbo:submit-start->spinner#submitStart turbo:submit-end->spinner#submitEnd"
+              }
+            ) do |form| %>
+          <div class="grid gap-4">
+            <div class="form-field">
+              <%= form.label :new_project_id, t(".new_project_id") %>
+              <%= form.collection_select(:new_project_id, @projects, :id, :full_path) %>
+            </div>
+            <div class="hidden" data-infinite-scroll-target="all"></div>
+            <div>
+              <%= form.submit t(".submit_button"),
+                          class: "button button--size-default button--state-primary",
+                          disabled: @projects.count.zero?,
+                          data: {
+                            action: "click->infinite-scroll#clear",
+                            "spinner-target": "submit",
+                            "turbo-submits-with": t(:".loading"),
+                          } %>
+            </div>
           </div>
-        </div>
+        <% end %>
       <% end %>
-    <% end %>
+    </div>
+
+    <div
+      role="status"
+      id="spinner"
+      class="
+        hidden absolute -translate-x-1/2 -translate-y-1/2 top-2/4 left-1/2
+      "
+    >
+      <svg
```

Comment:
**nitpick:** Don't use the svg here, take a look at how I have done this already in `/app/views/workflow_executions/submissions/_pipeline_selection_dialog.html.erb`

Reply:
67d22425affcf6dbbdecebd7293214f4693ae079